src/main/java/bio/terra/cli/command/resource/Mount.java

```diff
@@ -28,28 +29,46 @@ public class Mount extends BaseCommand {
   private Boolean disableCache;
 
   /**
-   * For controlled resources created by the calling user, the mount is always read-write regardless
-   * of the value of this flag.
+   * Whether to mount the resource(s) as read-only or read-write, overriding default mount
+   * permissions.
+   *
+   * <p>By default, for controlled resources created by the calling user, the mount is always
```

Comment:
"regardless of the value of this flag" makes it sound like this is ignored for controlled resources creating by the calling user. If I mount a resource I've created with `--read-only=true`, will that actually mount it read-write?

Reply:
This contradicts the top line and does correctly describe the actual behavior. `--read-only=true` will mount with only read perm. Updating the comment.